ci-operator/step-registry/cucushift/installer/check/vsphere/zone/cucushift-installer-check-vsphere-zone-commands.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+source "${SHARED_DIR}/govc.sh"
+source "${SHARED_DIR}/vsphere_context.sh"
+INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
+set -x
```

Comment:
Do we still need this option?

Reply:
if no other changes, i will delete later.